Application/EdFi.Ods.Admin.Tests/EdFi.Ods.Admin.Tests.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net48</TargetFramework>
+    <TargetFrameworks>net48</TargetFrameworks>
```

Comment:
Should this be dual targeting netcoreapp3.1?

Reply:
we cant target yet until we have converted the base project. that is on a separate ticket. (ODS-3505)